### PR TITLE
Bump golang deps to latest commits

### PR DESCRIPTION
**Pull request by ReToCode**
Fixes JIRA https://issues.redhat.com/browse/SRVKS-1152 and https://issues.redhat.com/browse/SRVKS-1113

## Proposed Changes
- Bumps golang deps to include https://github.com/knative/operator/pull/1577

### CODE REVIEW

**Comment by ReToCode** [comment]
@pierDipi I hope the eventing changes are ok as well?

**Comment by pierDipi** [comment]
Yes, @ReToCode can you rerun `make generated-files`

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

**Comment by pierDipi** [comment]
```
06:58:06.270 INFO:    Ensure no knative serving pods running
06:58:06.271 DEBUG:   [[ $(oc get pods -n knative-serving --field-selector=status.phase!=Succeeded -o jsonpath='{.items}') != '[]' ]] : Waiting until non-zero (max 600 sec.)
.........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
07:08:52.682 ERROR:   Time out of 600 exceeded
07:08:52.683 ERROR:   🚨 Error (code: 71) occurred at ./hack/lib/serverless.bash:23, with command: return 71
```

File: vendor/knative.dev/operator/pkg/reconciler/common/transformers.go [hunk 1]
```diff
@@ -28,8 +28,7 @@ func transformers(ctx context.Context, obj base.KComponent) []mf.Transformer {
 	return []mf.Transformer{
 		mf.InjectOwner(obj),
 		mf.InjectNamespace(obj.GetNamespace()),
-		JobTransform(obj),
```
Comment (pierDipi) [inline vendor/knative.dev/operator/pkg/reconciler/common/transformers.go:31]: This is missing now?

**Comment by pierDipi** [comment]
/lgtm

unhold once ready

**Pull request merged**